src/main/java/com/google/firebase/database/connection/NettyWebSocketClient.java

```diff
@@ -75,20 +80,27 @@
   public void connect() {
     checkState(channel == null, "channel already initialized");
     try {
-      TrustManagerFactory trustFactory = TrustManagerFactory.getInstance(
-          TrustManagerFactory.getDefaultAlgorithm());
-      trustFactory.init((KeyStore) null);
-      final SslContext sslContext = SslContextBuilder.forClient()
-          .trustManager(trustFactory).build();
       Bootstrap bootstrap = new Bootstrap();
+      SslContext sslContext = null;
+      if (this.isSecure) {
+        TrustManagerFactory trustFactory = null;
+        trustFactory = TrustManagerFactory.getInstance(
+            TrustManagerFactory.getDefaultAlgorithm());
```

Comment:
Huh? I think we can just say ` TrustManagerFactory trustFactory = TrustManagerFactory.getInstance(...)` instead of `null` here. WDYT?